birthday_automation/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
module name should be <shortcustomername>_<extention_modulename>_<short_feature_name>

Eg: janiclean_email_automation, or something down that line